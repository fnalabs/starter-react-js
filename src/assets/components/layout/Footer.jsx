import React from 'react'
import { Link } from 'react-router-dom'

export const Footer = () => (
  <footer>
    <nav>
      <h5>Sitemap</h5>
      <div><Link to='/'>Home</Link></div>
      <h5>Policies</h5>
      <div><Link to='/cookie'>Cookie</Link></div>
      <div><Link to='/privacy'>Privacy</Link></div>
    </nav>
  </footer>
)
