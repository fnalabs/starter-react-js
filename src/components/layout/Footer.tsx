import React, { FC } from 'react'
import Link from 'next/link'

const Footer: FC = () => (
  <footer>
    <nav>
      <h5>Sitemap</h5>
      <div><Link href='/'>Home</Link></div>
      <h5>Policies</h5>
      <div><Link href='/cookie'>Cookie</Link></div>
      <div><Link href='/privacy'>Privacy</Link></div>
    </nav>
  </footer>
)
export default Footer
