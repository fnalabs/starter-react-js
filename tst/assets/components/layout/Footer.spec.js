/* eslint-env jest */
import React from 'react'
import { MemoryRouter } from 'react-router'
import renderer from 'react-test-renderer'

import { Footer } from '../../../../src/assets/components/layout/Footer.jsx'

describe('<Footer />', () => {
  test('should render static content correctly', () => {
    const tree = renderer
      .create(<MemoryRouter><Footer /></MemoryRouter>)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
