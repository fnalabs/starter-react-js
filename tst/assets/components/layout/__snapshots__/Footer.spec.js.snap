// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render static content correctly 1`] = `
<footer>
  <nav>
    <h5>
      Sitemap
    </h5>
    <div>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </div>
    <h5>
      Policies
    </h5>
    <div>
      <a
        href="/cookie"
        onClick={[Function]}
      >
        Cookie
      </a>
    </div>
    <div>
      <a
        href="/privacy"
        onClick={[Function]}
      >
        Privacy
      </a>
    </div>
  </nav>
</footer>
`;
